elevenlabs/simple.py

```diff
@@ -72,3 +71,46 @@ def generate(
     else:
         assert isinstance(text, str)
         return TTS.generate(text, voice, model, api_key=api_key)
+
+
+async def agenerate(
+    text: Union[str, Iterator[str]],
+    api_key: Optional[str] = None,
+    voice: Union[str, Voice] = VOICES_CACHE[2],  # Bella
+    model: Union[str, Model] = "eleven_monolingual_v1",
+    stream: bool = False,
+    latency: int = 1,
+    stream_chunk_size: int = 2048,
+) -> Union[bytes, AsyncIterator[bytes]]:
```

Comment:
I'd suggest moving the majority of this code into a utility function since it is duplicated between the generate function